src/wallet/tx_builder.rs

```diff
@@ -303,6 +315,14 @@ impl<D: Database, Cs: CoinSelectionAlgorithm<D>> TxBuilder<D, Cs> {
             phantom: PhantomData,
         }
     }
+
+    /// Returns true if an absolute fee was specified
+    pub fn has_absolute_fee(&self) -> bool {
+        if self.fee_policy.is_none() {
+            return false;
+        };
+        matches!(self.fee_policy.as_ref().unwrap(), FeePolicy::FeeAmount(_))
```

Comment:
If you need to keep this code, it can probably be refactored to just:

```suggestion
        matches!(self.fee_policy, Some(FeePolicy::FeeAmount(_)))
```